src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -49,6 +51,8 @@
  * <li>
  * Ensures the first sentence ends with proper punctuation
  * (That is a period, question mark, or exclamation mark, by default).
+ * This is not for <code>@return</code>, since it automatically inserts
+ * a period after the text.
```

Comment:
```suggestion
 * Note that this check is not applied to inline <code>{@return ...}</code> tags,
 * because the Javadoc tools automatically appends a period to the end of the tag 
 * content.
```

Reply:
>I think we're oversimplifying things.

That's ok; not everyone is a java expert. I certainly didn't know about this nuance before this PR.

>people who will read the documentation are developers by the way

Yes, I am aware of this fact. 

>the below examples will show them how it exactly works

It is good to go into a bit more detail in the description like we have here, and mentioning some important detail of this check's behavior a couple of different ways is good; everyone fails to notice things in examples from time to time.